rules-java/api/pom.xml

```diff
@@ -16,9 +16,9 @@
     <dependencies>
         <!-- Local Dependencies -->
         <dependency>
-            <groupId>org.sonatype.tycho</groupId>
+            <groupId>org.eclipse.tycho</groupId>
```

Comment:
Does this still need to be here if we are including the Eclipse/Roaster deps in the windup-java-ast module? This and the following dependencies will create duplicate JARs in the project and increase the size (as well as lead to potential class compatibility issues).